Makefile

```diff
@@ -0,0 +1,18 @@
+TEST_IMAGE=specfile-tests
```

Comment:
Just a nitpick, this is not a shell script, so whitespace around `=` is allowed (and it would make this consistent with the two assignments below).